# service file name 'bug0_status.srv'


# Empty request

---

# target
geometry_msgs/Point target_position

# actual position
geometry_msgs/Point actual_position

# actual yaw
float32 actual_yaw

# actual status
int32 status

# active or not
bool active

# distance from the target
float32 distance

# target position reached
bool reached
